README.md

```diff
@@ -54,6 +53,23 @@ phoila install ibis-vega-transform
 phoila "examples/Charting Example.ipynb"
 ```
 
+### Tracing
+
+If you want to see traces of the interactiosn for debugging and performance analysis,
+install tthe `jaeger-all-in-one` binary and the `jupyterlab-server-proxy-saulshanabrook`
+lab extension to see the Jaeger icon in the launcher.
+
+
+Then open up the Jaeger tracer before running the notebook, run some visualizations and look at the traces.
```

Comment:
If we use the extension, all this should be automatic no?

Reply:
I will edit this to be more detailed.

Reply:
Does it make more sense now?